test/dev/fixtures/03-aurelia/package.json

```diff
@@ -8,53 +8,53 @@
   },
   "license": "MIT",
   "dependencies": {
-    "aurelia-bootstrapper": "2.3.2",
-    "aurelia-animator-css": "1.0.4"
+    "aurelia-animator-css": "1.0.4",
+    "aurelia-bootstrapper": "2.3.2"
   },
   "devDependencies": {
+    "@babel/core": "7.4.0",
+    "@babel/plugin-proposal-class-properties": "7.4.0",
+    "@babel/plugin-proposal-decorators": "7.4.0",
+    "@babel/preset-env": "7.4.2",
+    "@babel/register": "7.4.0",
     "aurelia-cli": "1.0.2",
+    "aurelia-loader-nodejs": "1.0.1",
+    "aurelia-pal-nodejs": "1.2.0",
     "aurelia-testing": "1.0.0",
     "aurelia-tools": "2.0.0",
-    "gulp": "4.0.0",
-    "minimatch": "3.0.4",
-    "through2": "3.0.0",
-    "vinyl-fs": "3.0.3",
-    "promise-polyfill": "8.1.0",
-    "regenerator-runtime": "0.13.2",
+    "aurelia-webpack-plugin": "3.0.0",
     "babel-eslint": "10.0.1",
-    "eslint": "latest",
+    "babel-jest": "24.5.0",
     "babel-loader": "8.0.5",
     "babel-plugin-istanbul": "5.1.1",
-    "@babel/plugin-proposal-decorators": "7.4.0",
-    "@babel/plugin-proposal-class-properties": "7.4.0",
-    "@babel/preset-env": "7.4.2",
-    "@babel/core": "7.4.0",
-    "@babel/register": "7.4.0",
-    "gulp-rename": "1.4.0",
-    "html-webpack-plugin": "3.2.0",
     "copy-webpack-plugin": "5.0.0",
-    "mini-css-extract-plugin": "0.4.3",
-    "aurelia-webpack-plugin": "3.0.0",
+    "css-loader": "1.0.0",
+    "del": "3.0.0",
     "duplicate-package-checker-webpack-plugin": "3.0.0",
-    "webpack": "4.27.0",
-    "webpack-cli": "3.1.2",
-    "webpack-dev-server": "3.1.9",
+    "eslint": "latest",
```

Comment:
Why is this latest? Should eslint be pinned to a specific version?

Reply:
@styfle we just went with the default from the aurelia generator